### ISSUE TITLE
Discount on app publication plans

### ISSUE THREAD

**Issue by bluesnap-rock** [Xtraball/Siberian#297]
Would be nice to be able to add multiple discounts to app publications. Would come in handy for regular users for getting, for example, 10%+ discount for publishing an app and also to resellers, giving them a top percentage off for publishing multiple apps.

**Comment by bbsniper** [comment]
Yes would be great to give resellers a 100% off voucher code and charge them off the platform

**Comment by userdevel** [comment]
@Slicit, @LikoV2: Any news after two year?
